src/Component/ItemList.js

```diff
@@ -1,13 +1,16 @@
 import React from "react"
+import '../index.css'
 
 const ItemList = ({ task, deleteItem, id }) => {
   return (
-    <div>
-      <p>{task}</p>
-      <button id={id} onClick={() => deleteItem(task)}>
-        {" "}
-        Delete
-      </button>
+    <div className="list">
+      <ul>
+
+        <li> <p>{task}</p>
+          <button className="btnDel" id={id} onClick={() => deleteItem(task)}> X </button>
```

Comment:
'btnDel' should be 'delBtn' or better still: 'deleteBtn' (its ok for names to be a bit longer if it makes it more clear what they are)